app/src/main/java/nl/tudelft/b_b_w/model/DatabaseHandler.java

```diff
@@ -195,6 +242,7 @@ public Block getBlock(String owner, String publicKey, int sequenceNumber) {
                 cursor.getString(5),
                 cursor.getString(6),
                 cursor.getInt(7));
+        block.setSeqNumberTo(sequenceNumber);
```

Comment:
@deepweeb @humanjas How to implement setting the correct seq no when it can also not be passed to the constructor?